FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -458,7 +464,15 @@ - (void)URLSession:(NSURLSession *)session
                                                              error:&dataError];
   NSString *targetTemplateVersion = _configFetch.templateVersionNumber;
   if (dataError == nil) {
-    targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    if ([response objectForKey:kTemplateVersionNumberKey]) {
+      targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    }
+    if ([response objectForKey:kIsFeatureDisabled]) {
+      gIsFeatureDisabled = [response objectForKey:kIsFeatureDisabled];
+      if (gIsFeatureDisabled) {
```

Comment:
can we add a test for this?

Reply:
also should we return a RealtimeError to the listener when realtime is disabled?

Reply:
Added a test for the flag and bubble error through listener